packages/p2p/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
Was this `.babelrc` custom made for P2P package's needs?

Reply:
yes, bcs the development is pretty similar to core/trader